src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTag.cs

```diff
@@ -29,22 +32,23 @@ namespace Microsoft.CodeAnalysis.Editor.InlineHints
     /// This is the tag which implements the IntraTextAdornmentTag and is meant to create the UIElements that get shown
     /// in the editor
     /// </summary>
-    internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
+    internal class InlineHintsTag : IntraTextAdornmentTag
     {
-        public const string TagId = "inline parameter name hints";
+        public const string TagId = "inline hints";
+
         private readonly IToolTipService _toolTipService;
         private readonly ITextView _textView;
         private readonly SnapshotSpan _span;
-        private readonly SymbolKey _key;
+        private readonly SymbolKey? _key;
```

Comment:
This goes away in https://github.com/dotnet/roslyn/pull/48479 when i update this UI code to not be C#/VB specific.